Bio/ExPASy/__init__.py

```diff
@@ -41,7 +40,8 @@ def get_prodoc_entry(
     For a non-existing key XXX, ExPASy returns an HTML-formatted page
     containing this text: 'There is currently no PROSITE entry for'
     """
-    return _binary_to_string_handle(urlopen("%s?%s" % (cgi, id)))
+    url = "%s?%s" % (cgi, id)
+    return _open(url)
```

Comment:
Personally I'd drop the local variable here, I find the one line version clear enough:

```python
return _open("%s?%s" % (cgi, id))
```

But we've had a similar discussion before, and this is essentially a matter of taste.

Reply:
I'm fine either way. Feel free to change it.

Reply:
Done in 29b467a2d8eac7b599a93c13ffd6239991026b67